tools/scripts/md-toc/gh-md-toc

```diff
@@ -0,0 +1,421 @@
+#!/usr/bin/env bash
+
```

Comment:
thought: :sweat_smile: Please never ask me to maintain this file.  Are we really sure this is worth it?

Reply:
You/we don't have to. I been using this for a few years now and has 3.2k stars. Is a keep it and forget it kind of thing IMO.

Worst case scenario, if something does break, we copy the new version with the fix and replace this script.

Reply:
It is copy-pasted as-is? No changes from the source?  Why have we copy pasted it?

Reply:
The readme pasted in suggests that we wouldn't need devs to manually download it (unless maybe it requires sudo?)

It also looks like the script doesn't work on windows:
> If you want it on Windows, you better to use a golang based implementation

Reply:
> The readme pasted in suggests that we wouldn't need devs to manually download it (unless maybe it requires sudo?)
> 
> It also looks like the script doesn't work on windows:
> 
> > If you want it on Windows, you better to use a golang based implementation
> 
> todo: Why don't we just use the go version instead of the mac/linux-only bash script, then we wont need to copy-paste code about?
> 
> `go install "github.com/ekalinin/github-markdown-toc.go/cmd/gh-md-toc@latest"` etc...

I happy to adhere to that suggestion if there is preference :) I just didn't think we cared much about windows dev tooling.

Reply:
I haven't used the go version of the tool personally before, I wonder if formatting and everything else is the same.

Reply:
I tried switching to the go project, but sadly the go project is missing the `insert` option, and only generates the table of content (i.e. doesn't update it automatically). So will prefer the current approach.

Reply:
@sourcenetwork/database-team Please chime in if you have any other suggestions regarding the copy-paste and lack of window dev tool support (or another tool). Happy to follow the consensus.

Reply:
If we never have to maintain this, I don't really care that we just copy-pasted it. I also don't think it's a big deal that it doesn't work on windows. Devs on windows can just manually edit if they have to. If someone what's to contribute to the go version to include insert, we can switch to the go version when it's supported.

Reply:
> If we never have to maintain this, I don't really care that we just copy-pasted it. I also don't think it's a big deal that it doesn't work on windows. Devs on windows can just manually edit if they have to. If someone what's to contribute to the go version to include insert, we can switch to the go version when it's supported.

I am okay with this! I have subscribed to the issue, if it gets implemented happy to migrate to using go version of the tool.